tests/base/utilities_01.cc

```diff
@@ -65,6 +65,18 @@ void test ()
                  ExcInternalError());
   }
 
+  {
+    const char *p = "alpha;; beta;; gamma ";
+    AssertThrow (Utilities::split_string_list (p, ";;").size() == 3,
+                 ExcInternalError());
+    AssertThrow (Utilities::split_string_list (p, ";;")[0] == "alpha",
+                 ExcInternalError());
+    AssertThrow (Utilities::split_string_list (p, ";;")[1] == "beta",
+                 ExcInternalError());
+    AssertThrow (Utilities::split_string_list (p, ";;")[2] == "gamma",
+                 ExcInternalError());
```

Comment:
Do you want me to change this and replace AssertThrows with ifs? In the test suite I thought this was what was basically done in tests.h... 

:)